utils/src/symbol-key.ts

```diff
@@ -0,0 +1 @@
+export const KBreadcrumbKey = Symbol('KBreadcrumbKey');
```

Comment:
```suggestion
export const KSymbolKey = (key: string) => Symbol(`${key}_KEY`);
```
Is it possible that other components will use it in the future? If it is possible to use it, then change